### DISCUSSION TITLE
Tailwind breaking styling on IOS devices

### DISCUSSION THREAD

**Discussion by PedroHenrique-F** [tailwindlabs/tailwindcss#7782] [Help]
Someone knows why im facing some issues with tailwind on mobile IOS devices? the styling is breaking a lot...  But on chromium browser is 100% working.

**Comment by skimhugo** [comment]
I think you can search keyword like  ios screen tailwind in google

**Comment by kiisi** [comment]
I am experiencing this issue too on iphone XR max
